lms/djangoapps/ccx/views.py

```diff
@@ -93,12 +94,14 @@ def wrapper(request, course_id):
 
         if not course.enable_ccx:
             raise Http404
-        # Disable ccx view if user is not allowed to create or manage ccx.
-        elif (configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and
+        # Disable ccx view for master courses if user is not allowed to create ccx.
+        elif (
+            not is_ccx_course(CourseKey.from_string(course_id)) and
```

Comment:
You can reuse this variable. @anfbermudezme 
```suggestion
            not ccx and
```